doc/README.md

```diff
@@ -1,15 +1,16 @@
-GAP has three main GAP manuals (they are called "main" to distinguish them from package 
+GAP has two main GAP manuals (they are called "main" to distinguish them from package 
```

Comment:
Not true any more - we have also HPC-GAP Reference Manual https://www.gap-system.org/Manuals/doc/hpc/chap0.html

Reply:
fixed by later changes

Reply:
will do